packages/next/src/index.ts

```diff
@@ -34,6 +34,19 @@ export default class LogtoClient {
       }
     });
 
+  handleSignOut = (redirectUri = this.config.baseUrl): NextApiHandler =>
+    this.withIronSession(async (request, response) => {
+      const nodeClient = this.createNodeClient(request);
+      await nodeClient.signOut(redirectUri);
+
+      request.session.destroy();
+      await this.storage?.save();
```

Comment:
nit: just want to check we are not handling any exceptions here right? just throw whatever happens.

Reply:
YES, will be handled in LOG-3748, the final solution is not decided yet.